src/classes/Frames.ts

```diff
@@ -0,0 +1,49 @@
+import { ErrorType } from "../types";
+
+export const getTFRFrames = async ({ sku }): Promise<ErrorType | void> => {
+  window.theFittingRoom?.getUserProfile().then(userProfile => {
+    console.log("userProfile: ", userProfile?.vto);
+    console.log("sku: ", sku);
+
+    const getBrandId = () => {
+      const brandStyleId = sku?.split('-')?.[0];
+      return String(brandStyleId || "");
+    }
+
+    const getUserFrames = () => {
+      let frames = [];
+
+      if (userProfile?.vto?.frames) {
+        frames = userProfile?.vto?.frames;
+      } else {
+        frames = userProfile?.vto?.[`${getBrandId()}`]?.[`${sku}`]?.frames;
+      }
+
+      return frames || [];
+    }
+
+  /*
+    const getColorWayId = () => {
+      const colorWayId = userProfile?.vto?.[`${getBrandId()}`]?.[`${sku}`]?.colorway_id;
+      return Number(colorWayId || null);
+    }
+  */
+
+    const getUserSku = () => {
```

Comment:
I'm having trouble understanding the intention of this function. It appears to retrieve the first sku in the list that exists for that brand right? There shouldn't ever be a time where anyone just wants to retrive the first sku in the list right? They would want to retrieve a specific sku wouldn't they?

`userSku` is also kind of an ambiguous name but we can probably decide on a better name once we clear up the purpose of the function.